src/App.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+
```

Comment:
It is definitely not too important in the tutorial scope and maybe you just did not noticed because of `git add --all` command from the tutorial, however I would really recommend not to commit changes that contain only new line in the middle (unless it is some fixes due to the code style).